core/src/test/java/io/confluent/rest/SslTest.java

```diff
@@ -293,6 +295,11 @@ private int makeGetRequest(String url, String clientKeystoreLocation, String cli
     CloseableHttpResponse response = null;
     try {
       response = httpclient.execute(httpget);
+
+      // Verify that the Jetty Server version header is not returned
```

Comment:
This is not an ideal place to be adding such test code. This is executed for each test in `SslTest` and this check has nothing to do with SSL.

Can I suggest you add a test to `RestApiTest.java` that explicitly tests just this one thing? e.g.

```java
  @Test
  public void shouldNotExposeServerVersionInfo() {
    // Given:
    final Client client = ClientBuilder.newBuilder().build();
    try {

      // When:
      final Response res = client
          .target(REST_APP.getHttpListener())
          .path("/info")
          .request(MediaType.APPLICATION_JSON_TYPE)
          .get();

      // Then:
      assertThat(res.getStatusInfo().getStatusCode(), is(HttpStatus.SC_OK));
      final String version = res.getHeaderString("Server");
      assertThat(version, is(nullValue()));

    } finally {
      client.close();
    }
  }
```

Reply:
Want to make sure the header is gone for both http and https, and SslTest is the only one that stands up https.   To put this in it's own test class (there is not pre-existing RestApiTest.java) the ssl setup would need to be duped / refactored to be shared with another test class.

I'll add a dedicated test method for this to SslTest.

Reply:
Duped and simplified the SslTest class, so there is now an ApiHeadersTest that tests both http and https.